src/govuk/components/input/template.test.js

```diff
@@ -382,4 +389,107 @@ describe('Input', () => {
       expect($component.attr('inputmode')).toEqual('decimal')
     })
   })
+
+  describe('when it includes a prefix', () => {
+    it('renders the input wrapper', () => {
+      const $ = render('input', examples['with prefix'])
+
+      const $wrapper = $('.govuk-form-group > .govuk-input__wrapper')
+      expect($wrapper.length).toBeTruthy()
+    })
+    it('renders the prefix inside the wrapper', () => {
+      const $ = render('input', examples['with prefix'])
+
+      const $prefix = $('.govuk-form-group > .govuk-input__wrapper > .govuk-input__prefix')
+      expect($prefix.length).toBeTruthy()
```

Comment:
Might be worth checking the prefix text is what we expect too

Reply:
In my I had covered this with `allows prefix text to be passed whilst escaping HTML entities` but I've added a new test to check the text in the default example, it's more explicit 👍